src/System.Configuration.ConfigurationManager/ref/System.Configuration.cs

```diff
@@ -1386,7 +1386,7 @@ public sealed partial class WhiteSpaceTrimStringConverter : System.Configuration
 }
 namespace System.Configuration.Internal
 {
-    public partial class DelegatingConfigHost : System.Configuration.Internal.IInternalConfigHost
+    public partial class DelegatingConfigHost : System.Configuration.Internal.IInternalConfigHost, System.Configuration.Internal.IInternalConfigHostPaths
```

Comment:
I'm good with the public members but I don't think it is necessary to include the internal interface. You can keep that in the baseline.

Reply:
actually it shouldn't need to be baselined given you make it internal in the implementation.